### PR TITLE
Document workaround for making zram mounts user-writable (c.f. #146)

### PR DESCRIPTION
**Pull request by eMPee584**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -56,6 +56,16 @@ mount-point = /var/tmp
...
 This will set up a /dev/zram1 with ext2 and generate a mount unit for /var/tmp.
 
+In case you want this path to be user-writable, you can use following
+"high-quality hack" until `systemd-makefs` provides a proper mechanism to
+set ownership of a generated filesystem. For the above example, create a
+`/etc/systemd/system/systemd-zram-setup@zram1.service.d/override.conf` file:
+
+```ini
+[Service]
+ExecStartPost=/bin/sh -c 'd=$(mktemp -d); mount "$1" "$d"; chmod 777 "$d"; umount "$d"; rmdir "$d"' _ /dev/%i
```
Comment (nabijaczleweli) [inline README.md:66]: /var/tmp (and /tmp) is *not* 777 (indeed, if you have anything public 777, that's a Problem).

They're 1777 – a=rwx,o+t.

File: README.md [hunk 2]
```diff
@@ -56,6 +56,16 @@ mount-point = /var/tmp
 
 This will set up a /dev/zram1 with ext2 and generate a mount unit for /var/tmp.
 
+In case you want this path to be user-writable, you can use following
+"high-quality hack" until `systemd-makefs` provides a proper mechanism to
+set ownership of a generated filesystem. For the above example, create a
+`/etc/systemd/system/systemd-zram-setup@zram1.service.d/override.conf` file:
```
Comment (nabijaczleweli) [inline README.md:62]: better to note `systemctl edit` here imo?

**Review by nabijaczleweli — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (eMPee584) [reply README.md]: Very good point, thanks. Had to refresh my sense of what that sticky bit is about..

**Comment by nabijaczleweli** [comment]
Merged as ac9477250221a671494f4cfc415819b965e30085, thanks!

**Comment by eMPee584** [comment]
.. 😯 .. did you cut the explanation from 1af60b2 intentionally or did you miss it? Thanks for merging anyways.

**Pull request closed without merging**